src/inmanta/agent/config.py

```diff
@@ -39,19 +39,6 @@
     Option("config", "agent-names", "$node-name",
            "Names of the agents this instance should deploy configuration for", is_str)
 
-agent_interval = \
```

Comment:
we can't just rename options, this is going to break stuff all over the place

1- change needs to be backward compatible if at all possible, existing config files should give the same result
2- needs to be in changelog 
3- we can deprecate, but than we need a big fat warning